dnainator-javafx/src/main/java/nl/tudelft/dnainator/javafx/controllers/FileOpenController.java

```diff
@@ -76,6 +91,7 @@ private void initialize() {
 		graphLoadService.setOnRunning(e -> progressDialog.show());
 		graphLoadService.setOnSucceeded(e -> {
 			graphProperty.setValue(graphLoadService.getValue());
+			done.setValue(!done.getValue());
```

Comment:
This is maybe done a little dirty, but right now I flip the property to enable loading new files after an existing one is loaded (since the property is notified, it will create a new split pane again).